### ISSUE TITLE
Suggestion: remove unnecessary spacing

### ISSUE THREAD

**Issue by Fazarel** [pmoreno-rodriguez/grav-theme-editorial#27]
/templates/blog.html.twig
/templates/item.html.twig

In order to remove space above title, when not using breadcrumbs

Changes from:
```
    <p>
    {% if show_breadcrumbs and config.plugins.breadcrumbs.enabled %}
        {% include 'partials/breadcrumbs.html.twig' %}
    {% endif %}
    </p>
```

To:
```
    {% if show_breadcrumbs and config.plugins.breadcrumbs.enabled %}
    <p>
        {% include 'partials/breadcrumbs.html.twig' %}
    </p>
    {% endif %}
```

**Comment by pmoreno-rodriguez** [comment]
@Fazarel,I  just applied your suggestion. In the next version of the theme it should already be applied.

**Comment by Fazarel** [comment]
After thinking about it, I think it would be better to simply delete the `<p>` and `</p>` that frame the breadcrumbs code snippet. This changes the spacing relative to the title in a different way between pages that have breadcrumbs enabled and those that don't. The theme loses homogeneity.

**Comment by pmoreno-rodriguez** [comment]
Fixed in version 2.0.0
